docs/6-the-headliner/3-advanced-deployments.md

```diff
@@ -1,29 +1,98 @@
 # Advanced Deployments
 
-
+Machine learning models don’t operate in isolation; they are often integrated into larger systems. With each release, it’s critical to ensure that existing integrations remain intact, the system’s performance is maintained or improved, and security and compliance requirements are met. Advanced deployment practices, such as canary releases, help minimize the risk of introducing regressions, reduce downtime, and enable confident rollouts of new versions.
 
 ## Canary Deployments
 
-Traffic splitting in test - promote one to prod
+Canary deployments, or A/B deployments, generally imply running two versions of the application at the same time for testing or experimentation purposes. Initially, you have one model handling all the requests. Then, a new version is deployed, and some of the traffic is routed to it. If the new version performs as expected, we gradually shift all traffic to it and retire the original model. However, if any issues arise with the new model, we can quickly revert all traffic back to the original model without the need for redeployment or additional waiting. Essentially, this <sup>[1](https://kserve.github.io/website/latest/modelserving/v1beta1/rollout/canary/)</sup>👇 
+
+![canary-diagram.png](./images/canary-diagram.png)
+
+KServe can distribute the traffic that coming to model endpoint. But how does it do it? KServe holds the information of each version of the models we deploy. By default, it sends 100% traffic to the latest deployed version. See this by running the following command in code-server terminal.
+
+  ```bash
+  oc get isvc jukebox -n <USER_NAME>-test
+  ```
+
+  You should get an output similar to this:
+
+  <div class="highlight" style="background: #f7f7f7; overflow-x: auto; padding: 10px;">
+  <pre><code class="language-bash">
+  NAME      URL                                                                          READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION       AGE
+  jukebox   https://jukebox-<USER_NAME>-test.<CLUSTER_DOMAIN>   True           100                              jukebox-predictor-00003   2d2h
+    </code></pre></div>
+
+    What this output tells you that the 100% of the traffic is going to the LATEST version.
```

Comment:
I think this one has a tab too much, if you remove one tab it will look like normal text instead of code.

Reply:
ah good catch!